ramda.js

```diff
@@ -4705,6 +4707,42 @@
     R.gte = op(function _gte(a, b) { return a >= b; });
 
 
+    function createMaxMin(comparator, initialVal) {
+        return function(list) {
+            if (arguments.length === 0) {
+                throw noArgsException();
+            }
+            var idx = -1, winner = initialVal, computed;
+            while (++idx < list.length) {
+                computed = +list[idx];
+                if (comparator(computed, winner)) {
+                    winner = computed;
+                }
+            }
+            return winner;
+        };
+    }
+
+    function createMaxMinWith(comparator) {
+        return function(valueComputer, list) {
+            if (!(list && list.length > 0)) {
+                return;
```

Comment:
Ok cool. Different strokes for different folks.

I started looking into this because of the Ramda benchmark comparing its `max` to `_.max`. However on further digging it appears to be comparing apples to oranges as `_.max`+`identity` is equiv to `_.max` and Ramda's is not.